libs/utils/environment/src/lib/static-environment.ts

```diff
@@ -0,0 +1,14 @@
+declare global {
+  interface Window {
+    ENV?: any
+  }
+}
+
+export const ENV_PREFIX = 'SI_PUBLIC_'
+
+export const getStaticEnv = (environmentVariableName: string) => {
+  if (!environmentVariableName.startsWith(ENV_PREFIX)) {
+    throw new Error(`Variable must be prefixed with ${ENV_PREFIX}`)
```

Comment:
Nice.

But I believe this also needs a documentation, most likely from the root README where the `<!-- environment placeholder -->` is also specified, since it won't be auto generated by nx.

Reply:
Good point, I added instructions on how to use this in the root README